base/special/trig.jl

```diff
@@ -1246,10 +1274,13 @@ end
 tand(x::Real) = sind(x) / cosd(x)
 
 """
-    sincosd(x)
+    sincosd(x::T) where T -> float(T)
 
 Simultaneously compute the sine and cosine of `x`, where `x` is in degrees.
 
+Throws a [`DomainError`](@ref) if `isinf(x)`, returns a `(T(NaN), T(NaN))` tuple if `isnan(x)`.
+Propagates `Missing`.
```

Comment:
Why is this the only function where `missing` propagation is documented?

Reply:
Because right below the docstring, the method handling that is defined. I didn't check the other ones, but I'll add to them as well.

Reply:
Here is the list of functions where `missing` seems to work as intended:

```
julia> @testset "all tests" begin
           oneretfuncs = (asind, acosd, atand, asecd, acscd, acotd, sind, cosd, tand, sin, cos, tan, sec, csc, cot, sinh, cosh, tanh, sech, csch, coth, asinh, acosh, atanh, asech, acsch, acoth)
           @testset "1-ret" begin
               @testset for f in oneretfuncs
                       @test propagatesMissing(f)
               end
           end
           @testset "2-ret" begin
               @testset for f in (sincos, sincosd)
                   @test f(missing) isa Tuple{Missing, Missing}
               end
           end
       end

....

Test Summary: | Pass  Error  Total  Time
all tests     |   11     18     29  0.8s
  1-ret       |    9     18     27  0.8s
    f = asind |           1      1  0.0s
    f = acosd |           1      1  0.0s
    f = atand |           1      1  0.0s
    f = asecd |           1      1  0.0s
    f = acscd |           1      1  0.0s
    f = acotd |           1      1  0.0s
    f = sind  |           1      1  0.0s
    f = cosd  |           1      1  0.0s
    f = tand  |           1      1  0.0s
    f = sin   |    1             1  0.0s
    f = cos   |    1             1  0.0s
    f = tan   |    1             1  0.0s
    f = sec   |           1      1  0.0s
    f = csc   |           1      1  0.0s
    f = cot   |           1      1  0.0s
    f = sinh  |    1             1  0.0s
    f = cosh  |    1             1  0.0s
    f = tanh  |    1             1  0.0s
    f = sech  |           1      1  0.0s
    f = csch  |           1      1  0.0s
    f = coth  |           1      1  0.0s
    f = asinh |    1             1  0.0s
    f = acosh |    1             1  0.0s
    f = atanh |    1             1  0.0s
    f = asech |           1      1  0.0s
    f = acsch |           1      1  0.0s
    f = acoth |           1      1  0.0s
  2-ret       |    2             2  0.0s
ERROR: Some tests did not pass: 11 passed, 0 failed, 18 errored, 0 broken.
```

Should I add docstrings to the ones that do work?

Reply:
@stevengj what's the desired way forward here?

Reply:
It seems like a good idea to document the behavior of functions that propagate `missing` indeed. But probably better say `missing` rather than `Missing`?

(I think it makes sense for `sincosd(missing)` to return `(missing, missing)`, as it's defined as returning `(sind(missing), cosd(missing))`. But anyway this cannot be changed.)

Reply:
The issue is that quite a lot of functions, per the output above, don't handle `missing` at all. Should I just leave them as is?